packages/opentelemetry-instrumentation-fetch/src/fetch.ts

```diff
@@ -145,6 +145,11 @@ export class FetchInstrumentation extends InstrumentationBase<
         this._getConfig().propagateTraceHeaderCorsUrls
       )
     ) {
+      const headers: Partial<Record<string, unknown>> = {};
+      api.propagation.inject(api.context.active(), headers);
+      if (Object.keys(headers).length > 0) {
+        api.diag.debug('headers inject skipped due to CORS policy');
```

Comment:
Minor suggestion on the actual error content ```headers not injected``` vs ```headers inject skipped```

And as a general diagnostic error logging (I know we have previously talked about adding a scope using a standard mechanism), but in the mean-time do we want to manually add the scope to make it easier for us and users to understand where this log cam from?

eg. (Using OT-Fetch as an example -- it probably needs a larger general discussion to define a general convention)
```api.diag.debug('OT-Fetch: headers not injected due to CORS policy");```

Reply:
I think this is yet to be decided how the namespace should be constructed but not in this PR, please move the discussion to separate issue. With regards to copy it is how it was suggested in the raised issue.